core/unit_tests/test_iterator.py

```diff
@@ -400,8 +400,6 @@ def test__get_next_page_response_new_no_token_in_response(self):
         iterator = self._makeOne(client, path, None)
         response = iterator._get_next_page_response()
         self.assertEqual(response['items'], [{'name': key1}, {'name': key2}])
-        self.assertEqual(iterator.page_number, 1)
-        self.assertEqual(iterator.next_page_token, token)
```

Comment:
Do we need assertions that thes values get updated in `_next_page` / `_http_page_iter` / `_wrap_gax`?